tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -67,6 +68,8 @@ public class EnvironmentHelper
                           : string.Empty;
         }
 
+        public int AgentPort { get; private set; }
```

Comment:
We should expose the port in `MockTracerAgent.Port` instead. `EnvironmentHelper` should not store this state. What happens if you create multiple mock agents, which port are we returning?

Reply:
Do we actually have multiple agents for any tests? Either way, point taken, I'll remove the property and have tests which want the port pull it from the agent.